configure.ac

```diff
@@ -2740,8 +2740,6 @@ AC_DEFINE_UNQUOTED([LDPD_SOCKET], ["$CFG_STATE%s%s/ldpd.sock"], [ldpd control so
 AC_DEFINE_UNQUOTED([ZEBRA_SERV_PATH], ["$CFG_STATE%s%s/zserv.api"], [zebra api socket])
 AC_DEFINE_UNQUOTED([BFDD_CONTROL_SOCKET], ["$CFG_STATE%s%s/bfdd.sock"], [bfdd control socket])
 AC_DEFINE_UNQUOTED([OSPFD_GR_STATE], ["$CFG_STATE%s/ospfd-gr.json"], [ospfd GR state information])
-AC_DEFINE_UNQUOTED([MGMTD_FE_SERVER_PATH], ["$CFG_STATE/mgmtd_fe.sock"], [mgmtd frontend server socket])
```

Comment:
This looks like a genuine bug fix for mgmtd ( and should be backported to 9.0 and 9.1 as well right? )  Should this be in it's own PR?

Reply:
TBH I'm still pondering if we shouldn't just backport this entire PR; saving state to `/var/run` is such a weird thing to do. But fixing that needs the entire other pile of commits too, which is a bit much…

… and because we have (historically) overloaded `--localstatedir` in this stupid way I can't really "undo" or split off that change for a backport, without that /var/lib is kinda "inaccessible" …

Reply:
Case in point: if you configure OSPFv3 with authentication trailers and then reboot your box… your sequence number is gone and all the other routers refuse you into the network due to replay protection :clown_face:

Reply:
(regardless, it's not going to be a clean backport ⇒ need manual backport PR anyways ⇒ can adjust/drop things if needed)